index.html

```diff
@@ -299,16 +453,124 @@ <h1>
                         <a href="#">ADD ADDRESS</a>
                     </p>
                 </div>
+                <div>
+                    <fieldset>
+                        <legend>Delivery Type</legend>
+                        <div>
+                            <label>
+                                <input type="radio" name="d-type" value="urgent"> Urgent 1-2 days
+                            </label>
+                        </div>
+                        <div>
+                            <label>
+                                <input type="radio" name="d-type" value="normal"> Normal 5-7 days
+                            </label>
+                        </div>
+                    </fieldset>
+                </div>
+                <div>
+                    <label>
+                        <input type="checkbox"> I agree to
+                        <a href="#">Terms and Conditions</a>.
+                    </label>
+                </div>
                 <div>
                     <button class="primary">Submit</button>
                     <button>Cancel</button>
                 </div>
             </div>
         </div>
+        <div>
+            <h1>Custom variables</h1> 
+            <div>
+                <p>Underwear.css uses CSS Custom Properties (CSS Variables) that you can configure. You can also use them by yourself, to adhere to the consistent look and feel.</p>
+                <table>
+                    <tr><th>Name</th><th>Default value</th><th>Sample</th></tr>
+                    <tr>
+                        <td><code>--default-font-size</code></td>
+                        <td><code>14px</code></td>
```

Comment:
I don't think it is sustainable to include the values as static text here. Perhaps you can populate it with a script?

Reply:
Done